packages/mongo-nav/src/MongoNav.tsx

```diff
@@ -206,7 +206,7 @@ function MongoNav({
   const currentOrgId = data?.currentOrganization?.orgId;
   const currentProjectId = data?.currentProject?.projectId;
 
-  const defaultURLS: Required<Omit<URLSInterface, 'userMenu'>> = {
+  const defaultURLS: Required<Omit<URLSDefaults, 'userMenu'>> = {
```

Comment:
Do we still need `Required` here? Seems like that's now supported via `URLSDefaults`. Ditto below too.

Reply:
👍